### PR TITLE
iOS project setup

### PR DESCRIPTION
**Pull request by TomoLV**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,24 +48,24 @@ _ios_job_template: &ios_job_template
...
 matrix:
   include:
     - <<: *android_job_template
       name: Build Android
       stage: native
-# Native build is turned off at the moment because of a certificate issue
-#    - <<: *ios_job_template
-#      name: Build iOS
-#      stage: native
+ Native build is turned off at the moment because of a certificate issue
```
Comment (mikolak) [inline .travis.yml]: Remove 😉

File: ios/Classes/Bridging/DartMethodCaller.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #import <Flutter/Flutter.h>
-#import "CommonTypes.h"
+#import "CommonTypes1.h"
```
Comment (mikolak) [inline ios/Classes/Bridging/DartMethodCaller.h]: BlemulatorCommonTypes?

File: ios/Classes/Error/BleError.h [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #import "BleErrorCode.h"
-#import "CommonTypes.h"
+#import "CommonTypes1.h"
```
Comment (mikolak) [inline ios/Classes/Error/BleError.h]: BlemulatorCommonTypes?

File: ios/Classes/Error/BleError.m [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #import "BleError.h"
-#import "JSONStringifier.h"
+#import "JSONStringifier1.h"
```
Comment (mikolak) [inline ios/Classes/Error/BleError.m]: BlemulatorJSONStringifier?

File: ios/Classes/Model/Characteristic.m [hunk 5]
```diff
@@ -1,5 +1,5 @@
 #import "Characteristic.h"
-#import "CharacteristicResponse.h"
+#import "CharacteristicResponse1.h"
```
Comment (mikolak) [inline ios/Classes/Model/Characteristic.m]: BlemulatorCharacteristicResponse?

File: ios/Classes/Model/Peripheral.m [hunk 6]
```diff
@@ -1,5 +1,5 @@
 #import "Peripheral.h"
-#import "PeripheralResponse.h"
+#import "PeripheralResponse1.h"
```
Comment (mikolak) [inline ios/Classes/Model/Peripheral.m]: BlemulatorPeripheralResponse?

**Review by mikolak — CHANGES_REQUESTED** [review]

File: ios/Classes/Bridging/DartMethodCaller.h (hunk 2 above)
Comment (TomoLV) [reply ios/Classes/Bridging/DartMethodCaller.h]: Changed according to the review suggestion.

File: ios/Classes/Error/BleError.h (hunk 3 above)
Comment (TomoLV) [reply ios/Classes/Error/BleError.h]: Changed according to the review suggestion.

File: ios/Classes/Error/BleError.m (hunk 4 above)
Comment (TomoLV) [reply ios/Classes/Error/BleError.m]: Changed according to the review suggestion.

File: ios/Classes/Model/Characteristic.m (hunk 5 above)
Comment (TomoLV) [reply ios/Classes/Model/Characteristic.m]: Changed according to the review suggestion.

File: ios/Classes/Model/Peripheral.m (hunk 6 above)
Comment (TomoLV) [reply ios/Classes/Model/Peripheral.m]: Changed according to the review suggestion.

**Review by mikolak — DISMISSED** [review]

**Review by mikolak — APPROVED** [review]

**Pull request merged**